src/style.css

```diff
@@ -1 +1,63 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+.search-bars {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  margin-top: 20px;
```

Comment:
You are adding both top and bottom margins. Be consistent with your margins (Add only top or bottom, but not both) to avoid potential margin collapse.